contribute.md

```diff
@@ -65,13 +65,13 @@ order. Link to the appropriate graphic rather than duplicating it.
 To reference image files in GitHub, use the following markup:
 
     {% figure [caption:"<caption information>"] [class:"image bg-light border"] %}
-    ![]({{site.url}}<relative-path>){:width="100%"}
+    ![]({{site.url}}/assets/images/<relative-path>){:width="100%"}
```

Comment:
Remove the `/` after `{{site.url}}`:
```
![]({{site.url}}assets/images/<relative-path>){:width="100%"}
```

Reply:
Based on my testing, it does not work without that slash.

Reply:
See https://github.com/rackerlabs/design-system/pull/265

Reply:
You're right; the CDN variable requires no slash, but the regular site URL variable does. Dismissing review.